database/migrations/2019_02_10_220000_add_dates_to_fdb.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class AddDatesToFdb extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('ports_fdb', function (Blueprint $table) {
+            $table->timestamp('date_discovered')->nullable();
```

Comment:
You should use the standard Laravel "timestamps" https://laravel.com/docs/5.7/migrations

Reply:
Correct, I ended up with this one instead of $table->timestamps(); so that I could change the names. But the DB structure is actually the same.
 
It is pretty simple to change now so let's go.

Reply:
You get niceties like Eloquent updating the timestamps automatically.

Reply:
In fact, we need to use Eloquent "create" and "update" for the auto-update of the timestamps. But at least it is now ready for these niceties :)